pytest_splunk_addon/standard_lib/cim_tests/test_templates.py

```diff
@@ -70,8 +70,7 @@ def test_cim_required_fields(
         cim_fields = splunk_searchtime_cim_fields["fields"]
         cim_tag_stanza = splunk_searchtime_cim_fields["tag_stanza"]
 
-        cim_fields_str = ', '.join(map(str,cim_fields))
-        cim_fields_type= ', '.join(map(lambda f:f.get_type(),cim_fields))
```

Comment:
There should not be any changes in test_template.py.  @zahrasidhpuri-crest  Can you confirm that this is not changed?

Reply:
Agreed, the change is not related to the task. But as previously the base-branch was develop, it had merge-conflicts in this file. Hence I resolved it.
Link to develop: https://github.com/splunk/pytest-splunk-addon/blob/develop/pytest_splunk_addon/standard_lib/cim_tests/test_templates.py#L73